### PR TITLE
Udpdate the schema documentation

### PR DESCRIPTION
**Pull request by JulienPeloton**
This PR brings updates in the Schema model due to recent change (#36). Note that this PR only focuses on the change for `Janusgraph Schema Model`, but we need to revisit also other sections (vertices, edges, Data Insertion Algorithm, ...). The other changes will be done in another PR.

### CODE REVIEW

File: docs/Schema-Model.md [hunk 1]
```diff
@@ -8,128 +8,132 @@ There are currently 4 column families in the main HBase table:
...
+ i:szmag1                         timestamp=1598256131226, value=float
+ i:szmag2                         timestamp=1598256131226, value=float
+ i:szmag3                         timestamp=1598256131226, value=float
+ i:tblid                          timestamp=1598256131226, value=long
+ i:tooflag                        timestamp=1598256131226, value=integer
+ i:xpos                           timestamp=1598256131226, value=float
+ i:ypos                           timestamp=1598256131226, value=float
+ i:zpclrcov                       timestamp=1598256131226, value=float
+ i:zpmed                          timestamp=1598256131226, value=float
+1 row(s)
```
Comment (saucam) [inline docs/Schema-Model.md]: probaBly remove these

File: docs/Schema-Model.md [hunk 2]
```diff
@@ -139,19 +143,17 @@ Note that this schema is subject to evolution. The above row will be translated
...
+    - `random_forest_snia`: returns the probability of an alert to be a SNe Ia using a Random Forest Classifier (binary classification)
+    - `snn`: returns the probability of an alert to be a SNe Ia using [SuperNNova](https://github.com/supernnova/SuperNNova). Two pre-trained models:
+      - `snn_snia_vs_nonia`: Ia vs core-collapse SNe
+      - `snn_sn_vs_all`: SNe vs. anything else (variable stars and other categories in the training)
+    - `microlensing`: returns the predicted class (among microlensing, variable star, cataclysmic event, and constant event) & probability of an alert to be a microlensing event in each band using [LIA](https://github.com/dgodinez77/LIA).
+    - `asteroids`: Determine if the alert is an asteroid (experimental).
+    - `nalerthist`: Number of detections contained in each alert (current+history). Upper limits are not taken into account.
+
+
+**TODO: describe TwoModeClassifier and SameValueClassifier logic.**
```
Comment (saucam) [inline docs/Schema-Model.md]: These are described here: 

https://github.com/astrolabsoftware/grafink/blob/master/docs/classifiers/VertexClassifiers.md

we probably need to update this file now

File: docs/Schema-Model.md (hunk 1 above)
Comment (JulienPeloton) [reply docs/Schema-Model.md]: Do you mean all the code block?

Comment (saucam) [reply docs/Schema-Model.md]: No i meant 1 row(s)

Comment (JulienPeloton) [reply docs/Schema-Model.md]: OK, done.

File: docs/Schema-Model.md (hunk 2 above)
Comment (JulienPeloton) [reply docs/Schema-Model.md]: Updated. Do you want to further update?

**Pull request merged**